cosmo_tester/test_suites/snapshots/snapshot_upgrade_test.py

```diff
@@ -126,43 +134,31 @@ def test_restore_snapshot_and_agents_upgrade(
     cfy.snapshots.list()
 
     logger.info('Restoring snapshot on latest manager..')
-    restore_execution = manager2.client.snapshots.restore(
-        snapshot_id,
-        tenant_name=manager1.restore_tenant_name,
-        )
+    restore_execution = manager1.client.snapshots.restore(snapshot_id)
     logger.info('Snapshot restore execution:%s%s',
                 os.linesep,
                 json.dumps(restore_execution, indent=2))
 
     cfy.executions.list(['--include-system-workflows'])
 
     restore_execution = wait_for_execution(
-        manager2.client,
+        manager1.client,
         restore_execution,
         logger)
     assert restore_execution.status == 'terminated'
 
     cfy.executions.list(['--include-system-workflows'])
 
-    manager2.use(tenant=manager1.restore_tenant_name)
-    client = create_rest_client(
-        manager2.ip_address,
-        username=cluster._attributes.cloudify_username,
-        password=cluster._attributes.cloudify_password,
-        tenant=manager1.tenant_name,
-        api_version=manager2.api_version,
-        )
-
     cfy.deployments.list()
```

Comment:
No, no, this is not right, don't worry- you're right, you still have the use there, I missed it in the diff!